docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -25,10 +25,15 @@ properties in the file:
 .. code-block:: text
 
     connector.name=oracle
-    connection-url=jdbc:oracle:thin:@example.net:1521/ORCLCDB
+    connection-url=jdbc:oracle:thin:@//example.net:1521/ORCLCDB
```

Comment:
Both are correct. Depends on how you want to connect to Oracle and what version of Oracle you are on.

Quoting the link you shared:

> Starting Oracle Database 10g, Oracle Service IDs are not supported.

This is the one we use in our existing example.

Oracle also supports a lot of other ways (all mentioned in the linked document) but it's important to note:

> Thin-style Service Name Syntax
> `jdbc:oracle:thin:scott/tiger@//myhost:1521/myservicename`

This is the one you are adding now.

So IMO somehow we should make it more obvious that the example is just that - an example. People should consult the documentation to see whatever URL they should use.

Reply:
My understanding of documentation (https://docs.oracle.com/cd/B28359_01/java.111/b31224/urls.htm#BEIDHCBA) is that these are correct forms:
- `connection-url=jdbc:oracle:thin:@example.net:1521:ORCLCDB`
- `connection-url=jdbc:oracle:thin:@//example.net:1521/ORCLCDB`

`connection-url=jdbc:oracle:thin:@example.net:1521/ORCLCDB` is **not** correct

Reply:
But 12.x documentation here https://docs.oracle.com/en/database/oracle/oracle-database/21/tdpjd/getting-started.html#GUID-D1652CF5-5A80-40BF-BB96-016D1694BE5A gives example:
```
Connection connection =
        DriverManager.getConnection("jdbc:oracle:thin:@localhost:1522/xepdb1", "hr", "hr");
```
So I guess all forms are correct...

Reply:
Given all that we should probably have multiple examples and links to the JDBC driver docs and explain that the URL just has to be valid for the driver use and the connected Oracle database from Trino..

Reply:
Followup question, the requirements section above says that you need Oracle 12 or higher, so should we even be documenting v11?

Reply:
We should not imho

Reply:
@mosabua The existing example is present in 12g documentation too - the last link that Pawel shared.

We can have multiple examples but it can't possibly be exhaustive. If left to me I'd just add a comment in the example config (before the `connection-url` line) that says to consult the Oracle documentation for the URL to use.

Reply:
Updated, please have a look to see if that's a good compromise.

Reply:
What changed?

Reply:
I remembered how to use git, apparently. 😅

Changes are there now.